patches/react-native-screens+3.29.0+001+initial.patch

```diff
@@ -0,0 +1,49 @@
+diff --git a/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt b/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt
+index d9e0e58..2d946c4 100644
+--- a/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt
++++ b/node_modules/react-native-screens/android/src/fabric/java/com/swmansion/rnscreens/FabricEnabledViewGroup.kt
+@@ -3,7 +3,6 @@ package com.swmansion.rnscreens
+ import android.view.ViewGroup
+ import androidx.annotation.UiThread
+ import com.facebook.react.bridge.ReactContext
+-import com.facebook.react.bridge.ReadableMap
+ import com.facebook.react.bridge.WritableMap
+ import com.facebook.react.bridge.WritableNativeMap
+ import com.facebook.react.uimanager.FabricViewStateManager
+@@ -13,6 +12,9 @@ import kotlin.math.abs
+ abstract class FabricEnabledViewGroup constructor(context: ReactContext?) : ViewGroup(context), FabricViewStateManager.HasFabricViewStateManager {
+     private val mFabricViewStateManager: FabricViewStateManager = FabricViewStateManager()
+ 
++    private var lastSetWidth = 0f
```

Comment:
This change was [merged upstream](https://github.com/software-mansion/react-native-screens/pull/2024/files) on 2024-02-08 and released in 3.30.0. Let's create a separate PR to upgrade this and remove the dependency.

Reply:
Here is the PR: https://github.com/Expensify/App/pull/40009